cfme/containers/provider/__init__.py

```diff
@@ -83,6 +87,7 @@
 
 @BaseProvider.add_base_type
 class Provider(BaseProvider, Pretty):
+    appliance = CurrentAppliance()
```

Comment:
CurrentAppliance gives the same as get_or_create_current_appliance. I guess this call can be dropped.